chirp/drivers/kg935g.py

```diff
@@ -941,23 +955,14 @@ def _get_settings(self):
                               "Call Name "+callnum, val)
             call_grp.append(rs)
 
-            _codeobj = eval("_call.call_code_"+callnum)
-            _code = "".join([callchars[x] for x in _codeobj if int(x) < 0x0A])
-            val = RadioSettingValueString(3, 6, _code, False)
-            val.set_charset(callchars)
-            rs = RadioSetting("call_code_"+callnum,
-                              "Call Code " + callnum, val)
-
-            def apply_call_code(setting, obj):
-                value = []
-                for j in range(0, 6):
-                    try:
-                        value.append(callchars.index(str(setting.value)[j]))
-                    except IndexError:
-                        value.append(0xFF)
-                obj.call_code = value
-            rs.set_apply_callback(apply_call_code,
-                                  eval("_call.call_code_"+callnum))
+            x = i-1
```

Comment:
Spaces around the `-` please. Our style checker doesn't get these, but most do, so when I see them, I try to point them out for the future.

Reply:
ok - this passed the tox style check --- must be a different set of rule checks.  will fix.

Reply:
Yeah, that's why I'm saying ours does, but it's too loose and I want to tighten it up, so I just try to spot these when I can to reduce the number of things we'll have to fix later. Not a huge deal, but if you're revising, might as well.